docs/web_reference.rst

```diff
@@ -830,6 +831,10 @@ Response
    :param str charset: response's charset. ``'utf-8'`` if *text* is
                        passed also, ``None`` otherwise.
 
+   :param int zlib_executor_size: length in bytes which will trigger zlib compression
+                            of body to happen in an executor
+
```

Comment:
Please insert `.. versionadded:: 3.5` directive here.

Reply:
aiohttp `3.5.0` was not released yet, only `3.5.0 alpha 0`.
So please use just `3.5`, the last zero should be omitted.